playbook.yml

```diff
@@ -1,5 +1,5 @@
 - name: Deploy businesscard on OpenShift
   hosts: all
-  gather_facts: no
+  gather_facts: yes
```

Comment:
Can't find where this is needed. Would you mind point it to me?

Reply:
It's necessary to access the variable `ansible_date_time.iso8601` in the "App Restart" task:
https://github.com/epfl-si/businesscard.ops/blob/72c2e833c01f62a3bc6040facc7522065600a75c/roles/businesscard-openshift/tasks/app.yml#L225C33-L225C58

Reply:
Initially, gather_facts was enabled to get `{{ ansible_date_time.iso8601 }}` in the 'App Restart' task. 
However, I switched to using the date command instead, to avoid unnecessary fact gathering and improve playbook efficiency.